crypto/evp/pmeth_lib.c

```diff
@@ -686,6 +760,52 @@ static int legacy_ctrl_str_to_param(EVP_PKEY_CTX *ctx, const char *name,
         return ret;
     }
 
+    if (strcmp(name, "rsa_padding_mode") == 0) {
```

Comment:
At the very least, I would like to see aliases with `rsa_` removed.

Reply:
These are legacy ctrls - so I'm not sure why we would add new aliases for them?

Reply:
Because `-pkeyopt` keys will eventually be directly used as OSSL_PARAM keys, and we might as well allow the legacy controls to be accessed with similar names (plus it's a pet peeve of mine...  I have never understood why it couldn't simply be "mode", for example)

Reply:
Any more thoughts on this?